pkgs/development/python-modules/vapoursynth/default.nix

```
@@ -1,4 +1,4 @@
-{ vapoursynth, cython, buildPythonPackage, unittestCheckHook }:
+{ vapoursynth, cython, buildPythonPackage, unittestCheckHook, pythonPackages }:
```

```suggestion
{ vapoursynth, cython, buildPythonPackage, unittestCheckHook, python }:
```

to make sure that we use the same python version